tests/test_pep517.py

```diff
@@ -54,9 +54,19 @@ def run(cmd: List[str], *args: object, **kwargs: object) -> subprocess.Completed
 
 
 def test_invalid_config_settings(package_pure, tmp_dir_session):
-    raises_error = pytest.raises(mesonpy.ConfigError, match='Unknown config setting: invalid')
+    raises_error = pytest.raises(mesonpy.ConfigError, match="Unknown config setting: 'invalid'")
 
     with raises_error:
         mesonpy.build_sdist(tmp_dir_session, {'invalid': ()})
     with raises_error:
         mesonpy.build_wheel(tmp_dir_session, {'invalid': ()})
+
+
+def test_invalid_config_settings_suggest(package_pure, tmp_dir_session):
+    raises_error = pytest.raises(mesonpy.ConfigError, match='setup-args')
```

Comment:
nitpick: Can you put the full message here?

Reply:
I opted to only search for the target I think we should have hit because if the number of options does grow as @henryiii is worried it will I expect that this message will change in a way that should not fail.  I changed the pattern to check more of the message.

Reply:
Thanks!